client/components/calypso-i18n-provider/index.tsx

```diff
@@ -25,7 +27,11 @@ const CalypsoI18nProvider: React.FunctionComponent = ( { children } ) => {
 		};
 	}, [] );
 
-	return <I18nProvider localeData={ localeData }>{ children }</I18nProvider>;
+	return (
+		<LocaleProvider localeSlug={ localeSlug || 'en' }>
+			<I18nProvider localeData={ localeData }>{ children }</I18nProvider>
+		</LocaleProvider>
+	);
```

Comment:
@p-jackson is this the right/expected way to use this provider?

Reply:
Yes this is it! In the Calypso world we should expect to have access to both the `react-i18n` hook _and_ the locale slug, so wrapping everything in both providers is the way to do it. Defaulting to `'en'` in the Calypso app also seems like the right thing, but it looks like `i18n-calpyso` already does that so maybe it's best to keep that logic there and not have the `|| 'en'` fallback.

Reply:
Thanks!

This en fallback was to make typescript happy with the getLocaleSlug new type exposed by i18n-calypso - that one I had to add recently - are you sure it does the fallback?